frontend/packages/dev-console/src/utils/pipeline-utils.ts

```diff
@@ -56,6 +56,18 @@ export const conditions = {
     task.runAfter && task.runAfter.length > 0,
 };
 
+export enum ListFilterIds {
+  Running = 'Running',
+  Failed = 'Failed',
+  Succeeded = 'Succeeded',
+}
+
+export enum ListFilterStatuses {
```

Comment:
Perhaps what we can do here is make a map instead of an enum. We could us the `ListFilterIds` as keys in the map so show they are related. Also, `ListFilterStatuses` is probably better named `ListFilterLabels` as they are labels.

Something like:
```
export const ListFilterLabels = {
  [ListFilterIds.Running]: 'Running',
  [ListFilterIds.Failed]: 'Failed',
  [ListFilterIds.Succeeded]: 'Complete', // this was the old label noted in a previous comment
};
```